sys/include/net/gnrc/mac/internal.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_mac  A common MAC type for providing key MAC parameters and helper functions
```

Comment:
`net_gnrc_mac` already gets defined in `mac.h`, you might want to use `@ingroup net_gnrc_mac` here.